tests/RobotFramework/tests/cumulocity/availability/heartbeat.robot

```diff
@@ -22,6 +22,10 @@ Heartbeat is sent
     ...    The tests therefore relies on the backend availability service which then sets the c8y_Availability fragment
     ...    based on the last received telemetry data.
     Device Should Have Fragment Values
+    ...    c8y_RequiredAvailability.responseInterval\=1m
```

Comment:
```suggestion
    ...    c8y_RequiredAvailability.responseInterval\=60
```

c8y accepts only seconds. At least, this caused the `Heartbeat is sent` test failure.

Reply:
This is wrong. Sorry! Corrected in https://github.com/thin-edge/thin-edge.io/pull/3174#discussion_r1796915753